Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -173,6 +173,15 @@ define([
             get : function() {
                 return this._readyPromise.promise;
             }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
```

Comment:
This should just be the same exact doc as `Cesium3DTileContent` (too bad we can't use `@inheritdoc` since it's not real inheritance).